### DISCUSSION TITLE
Connect to a Data Source (MySQL) via an SSH tunner

### DISCUSSION THREAD

**Discussion by mmick66** [mikro-orm/mikro-orm#6260] [Q&A] [answered]
I am wondering whether it's possible to open a connection with a data-source, in my case being an MySQL database, via an SSH tunnel that listens to port 22 and expects a username and password.

**Comment by B4nan** [comment, marked as answer]
We only wrap knex, which wraps mysql2, we don't handle connections here. Tried to find some docs on this on knex and they probably also don't support this natively. Either way, try searching how to do this with those packages, if its possible, I can try to help you doing it from here.

**Reply by mmick66** [reply]
I ended up opening the tunnel locally from my command like 
```
$ ssh -L 3306:localhost:3306 username@192.xxx.xxx.xx
```
Thanks!
